magento2.go

```diff
@@ -1,13 +1,32 @@
 package gocommerce
 
 import (
+	"encoding/json"
+	"errors"
 	"fmt"
+	"os"
+	"regexp"
 	"strconv"
 	"strings"
 
 	"github.com/sansecio/gocommerce/phpcfg"
 )
 
+type (
+	composerRoot struct {
+		Name    string            `json:"name"`
+		Version string            `json:"version"`
+		Require map[string]string `json:"require"`
+	}
+
+	composerPackages struct {
```

Comment:
Just FYI, if these are only for JSON unmarshalling, alternatively you can use [gjson](https://github.com/tidwall/gjson) which has jq-like query syntax.